### ISSUE TITLE
eslint 4.6.1 intellisense for indentation of new instances/functions parameters

### ISSUE THREAD

**Issue by danielserrao** [eslint/eslint#9292]
**Tell us about your environment**

* **ESLint Version:** 4.6.1
* **Node Version:** 6.11.2
* **npm Version:** 5.4.0

**What parser (default, Babel-ESLint, etc.) are you using?**
ESLint

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
---
  env:
    node: true
    es6: true
  extends: eslint:recommended
  rules:
    indent:
      - 1
      - 2
      - SwitchCase: 1
        MemberExpression: 0
        ObjectExpression: 1
        FunctionExpression: 
          parameters: 'first'
          body: 1
        FunctionDeclaration: 
          parameters: 'first'
          body: 1
```

</details>

**What did you do? Please include the actual source code causing the issue.**

```js
      const documentDBClient = new DocumentDBCollectionClient(
             'param1', //Should see a warning here
        'param2',
        'param3',
        'param4'
      );
```

**What did you expect to happen?**
I expected to see an intellisense warning on my code which don't happen and a warning on the console which happen.

**What actually happened? Please include the actual, raw output from ESLint.**
warning  Expected indentation of 8 spaces but found 13          indent

**Comments**

This also happens on the functions parameters such as

```js
    return executeRequest(
         'param1', //Should show an intellisense warning
      'param2',
      'param3'
    )
    .then(results => {
      //Do something
    });
```

**Comment by platinumazure** [comment]
Hi @danielserrao, thanks for the issue!

If you are seeing the warning show up when running ESLint directly in the console, but you aren't seeing it in your editor, that is most likely an issue with your editor integration.

What editor integration are you using? Would you be able to open up an issue on that integration's issue list?

**Comment by danielserrao** [comment]
Hi platinumazure!

I'm using Visual Studio Code 1.15.1 . Do you mean create the issue in another place? Which list you mean?

Thanks for the help.

**Comment by not-an-aardvark** [comment]
If the error appears on the command line but it doesn't appear in your VSCode integration, then this could be an issue with [vscode-eslint](https://github.com/microsoft/vscode-eslint), so you could report an issue there.
